readers/cassandra/messages_test.go

```diff
@@ -96,10 +99,24 @@ func TestReadAll(t *testing.T) {
 			limit:    5,
 			messages: messages[40:42],
 		},
+		"read message with non-existent subtopic": {
+			chanID:   chanID,
+			offset:   0,
+			limit:    msgsNum,
+			andQuery: map[string]string{"subtopic": "not-present"},
+			messages: []mainflux.Message{},
+		},
+		fmt.Sprintf("read message with subtopic: %s", subtopic): {
```

Comment:
Just specify if subtopic exists or not, not need for using `fmt.Sprintf` and specifying subtopic value.